v2/passwordless/serverless/with-aws-lambda/setup-api-gateway.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: setup-api-gateway
+title: Set up API Gateway
+hide_title: true
+---
+
+<!-- COPY SECTION -->
+<!-- ./thirdpartyemailpassword/serverless/with-aws-lambda/setup-api-gateway.mdx -->
+<!-- 1 -->
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import TabItem from '@theme/TabItem';
+
+# Set up API Gateway
+
+## 1) Create a REST API Gateway
+
+We will be using AWS API Gateway to create a REST API that will be used to communicate with our Lambda functions. 
+
+<img src="/img/integration-lambda/create-api-gateway.png"/>
+
+## 2) Set up auth routes
+
+Create a `/auth` resource and then `/auth/{proxy+}` resources. This will act as a catch-all for all supertokens auth routes. **Enable CORS** while creating the proxy resource. 
+
+<img src="/img/integration-lambda/create-proxy-route.png"/>
+
+<img src="/img/integration-lambda/route-creation-complete.png"/>
+
+## 3) Attach lambda to the `ANY` method of the proxy resource
+
+Click on the "ANY" method and then "Integration" to configure the lambda function. Check **Lambda proxy integration** and then select your lambda function.
+
+<img src="/img/integration-lambda/configure-lambda-integration.png"/>
+
+## 4) Enable CORS for the proxy path
+
+Click on the `{proxy+}` resource and then "Enable CORS" button to open the CORS configuration page. 
+
+<img src="/img/integration-lambda/click-enable-cors.png" />
+
+<AppInfoForm
+    askForWebsiteDomain
+    hideWebsiteBasePathField
+>
+
+- In the CORS configuration page do the following:
+    - Add `rid,fdi-version,anti-csrf,st-auth-mode` to the existing `Access-Control-Allow-Headers`
+    - Set `Access-Control-Allow-Origin` to `'^{form_websiteDomain}'`
```

Comment:
The '^{form_websiteDomain}' is not rendering correctly in the docs. You need to import `AppInfoForm` at the top of the file. I think you also need to enable the Default 4xx response to have these cors headers set.

This will require a change in the screenshot - maybe you can edit the image to add the same checkbox next to the 4xx text

Reply:
done